src/classes/UTIL_ACCT_Naming.cls

```diff
@@ -35,296 +35,647 @@
 */
 public class UTIL_ACCT_Naming {
     /*******************************************************************************************************
-    * @description The user defined Administrative Account record type.
+    * @description The default Account type.
     */
-    private static ID userDefinedAdminRecordTypeId = Util_Describe.getAdminAccRecTypeID();
+    private static ID defaultRecTypeId = UTIL_CustomSettingsFacade.getSettings().Account_Processor__c;
 
     /*******************************************************************************************************
-    * @description update admin account names if any changes made to contacts' lastname
-    * @param listContactsChangedLastName list of contacts whose lastnames are changed
-    * @return list<Account>
+    * @description The Household Account record type.
     */
-    public static list<Account> updateAdminAccountsName(List<Contact> listContactsChangedLastName) {
-        List<Account> listAccToUpdate = new List<Account>();
-        Map<Id, Contact> mapAccIdToCon = new Map<Id, Contact>();
-        for (Contact con : listContactsChangedLastName) {
-            mapAccIdToCon.put(con.AccountId, con);
+    private static ID hhAccountRecordTypeId = UTIL_Describe.getHhAccRecTypeID();
+
+    /*******************************************************************************************************
+    * @description The Administrative Account record type.
+    */
+    private static ID adminAccountRecordTypeId = UTIL_Describe.getAdminAccRecTypeID();
+
+    /*******************************************************************************************************
+    * @description The default/selected Household Account name format.
+    */
+    public static String hhNameFormat = UTIL_CustomSettingsFacade.getSettings().Household_Account_Naming_Format__c;
+
+    /*******************************************************************************************************
+    * @description The default/selected Administrative Account name format.
+    */
+    public static String aaNameFormat = UTIL_CustomSettingsFacade.getSettings().Admin_Account_Naming_Format__c;
+
+    /*******************************************************************************************************
+    * @description Update admin Account names if any changes made to Contacts' lastname.
+    * @param contactsChangedLastName list of Contacts whose LastNames have changed.
+    * @return List<Account>.
+    */
+    public static List<Account> updateAdminAccountsName(List<Contact> contactsChangedLastName) {
+        List<Account> accountToUpdate = new List<Account>();
```

Comment:
Nit: Prefer "accountsToUpdate" since it's a list and there will be potentially more than one account